### ISSUE TITLE
Narrator Issues with Bot Framework/Webchat

### ISSUE THREAD

**Issue by murcinar** [microsoft/BotFramework-WebChat#3572]
## Version

Microsoft Bot Framework v4:
Microsoft.Bot.Builder 4.6.2 Nuget package

## Describe the bug

while using Narrator with Microsoft WebChat Bot framework, we have identified the following issues:

*after posting a message, Narrator reads the conversation from the beginning which it already read the previous messages.
* Narrator announces "group" over and over again many times in many contexts irrelevantly, distracting the user. 
* After an answer is posted by the Bot, when there are suggested action buttons, they are not read by the narrator unless the user focuses on them by the tab key.

## Steps to reproduce

Launch a bot application created with the MS Bot Framework and narrate the conversation with Microsoft Narrator.

## Expected behavior

After the Narrator finishes the narration, I start conversing with the Bot and I would expect every post in the conversation to be read only once, in the right order, while when there are suggested actions available with an answer, they should be in the queue for the Narrator to be read without having to tab into them as visually impaired users won't even know that the suggested actions would be there. I would also expect the Narrator to narrate only the relevant information and not "group" over and over again.

## Additional context

[Bug]

**Comment by corinagum** [comment]
- What version of Web Chat are you using? Note that Web Chat and Bot Builder are non-coupled. If your version of WC is not 4.10.1, please update to that version and let us know if you are experiencing the same issues.
- What browser are you testing Narrator with? Right now the recommended browser is Microsoft Edge, so there may be some behavior differences if you're using Narrator with Chrome, for example. These kinds of differences are bugs related to the browser/AT, not Web Chat. Your first bullet sounds like you may be using Narrator with a different browser. For Chrome, we recommend using NVDA for a better user experience. 
- Your second bullet reminds me of the work done for Suggested Actions in 4.7: https://github.com/microsoft/BotFramework-WebChat/pull/2613. Hopefully this will also be addressed with a WC version bump.

**Comment by corinagum** [comment]
Closing due to inactivity. Please feel free to respond to the above questions and I will reopen.
